recipe/meta.yaml

```diff
@@ -1,15 +1,14 @@
 {% set name = "yarr" %}
-{% set version = "1.5.2" %}
+{% set version = "1.5.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://gitlab.cern.ch/yarr/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
-  sha256: 56dd0a42f5260470239f76b89aad016913e8267fbf3e10227abf1797618de18d
+  sha256: b3f72d4a469754be92d73487f3d6bdf1b573bffe5a26acb4d0d884db84da1bf6
```

Comment:
```suggestion
  sha256: a94a8e879bc7294c0b8389f740ce93344dc87a10e867baa0796dfeadeb2ff377
```

From the CI logs

```
RuntimeError: SHA256 mismatch for yarr-v1.5.3.tar.gz: obtained 'a94a8e879bc7294c0b8389f740ce93344dc87a10e867baa0796dfeadeb2ff377' != expected 'b3f72d4a469754be92d73487f3d6bdf1b573bffe5a26acb4d0d884db84da1bf6'
```

Reply:
The SHA you gave I'm sure was correct for how you generated it, but conda-forge uses `openssl sha256` to generate the sha for checking:

```console
$ curl -sL https://gitlab.cern.ch/yarr/yarr/-/archive/v1.5.3/yarr-v1.5.3.tar.gz | openssl sha256
SHA2-256(stdin)= a94a8e879bc7294c0b8389f740ce93344dc87a10e867baa0796dfeadeb2ff377
```

so you have to match that.